Branch-SDK/src/main/java/io/branch/referral/ServerRequestIdentifyUserRequest.java

```diff
@@ -53,6 +53,10 @@ public ServerRequestIdentifyUserRequest(Defines.RequestPath requestPath, JSONObj
 
     public void onRequestSucceeded(ServerResponse resp, Branch branch) {
         try {
+            if (getPost() != null && getPost().has(Defines.Jsonkey.Identity.getKey())) {
+                prefHelper_.setIdentity(Branch.installDeveloperId);
```

Comment:
If the call fails, the identity will be sent in install still but wont be set to prefhelper. That means v1/profile will be called again next time setIdentity is called.

For the identity response, do you mean in the ServerResponse from the code snippet or the identity from prefhelper?

Reply:
If the service response is equivalent to what got initially sent on install, since that's what's going to be saved to prefhelper and sent on subsequent events. So in other words, making sure that the `install developer id == all event developer id`

Reply:
Disregard the above, I misread the prefhelper line. But should still check that the response is equivalent

Reply:
After looking at the server response, it doesn't appear to actually include identity.

```
{
   "session_id":"1129134632836341318",
   "randomized_bundle_token":"532323229055121557",
   "link":"https:\/\/bnctestbed.test-app.link\/?%24randomized_bundle_token=532323229055121557",
   "link_click_id":"548712739951657893",
   "referring_data":"{\"$address_city\":\"city1\",\"$address_country\":\"Country1\",\"$address_postal_code\":\"postal_code\",\"$address_region\":\"Region1\",\"$address_street\":\"Street_name1\",\"$android_deeplink_path\":\"custom\/path\/*\",\"$android_passive_deepview\":\"branch_passive_default\",\"$canonical_identifier\":\"item\/12345\",\"$canonical_url\":\"https:\/\/branch.io\/deepviews\",\"$condition\":\"EXCELLENT\",\"$content_schema\":\"COMMERCE_PRODUCT\",\"$currency\":\"USD\",\"$exp_date\":\"1573415635000\",\"$identity_id\":\"532323229055121557\",\"$image_captions\":[\"my_img_caption1\",\"my_img_caption_2\"],\"$ios_url\":\"http:\/\/example.com\/ios\",\"$keywords\":[\"My_Keyword1\",\"My_Keyword2\"],\"$latitude\":12.07,\"$longitude\":-97.5,\"$match_duration\":100,\"$og_description\":\"my_product_description1\",\"$og_image_url\":\"https:\/\/test_img_url\",\"$og_title\":\"My Content Title\",\"$one_time_use\":false,\"$price\":23.2,\"$product_brand\":\"my_prod_Brand1\",\"$product_category\":\"Baby & Toddler\",\"$product_name\":\"my_product_name1\",\"$product_variant\":\"3T\",\"$publicly_indexable\":\"false\",\"$quantity\":2,\"$rating\":6,\"$rating_average\":5,\"$rating_count\":5,\"$rating_max\":7,\"$sku\":\"1994320302\",\"+click_timestamp\":1532061538,\"+referrer\":\"com.facebook.orca\",\"Custom_Content_metadata_key1\":\"Custom_Content_metadata_val1\",\"source\":\"android\",\"~campaign\":\"Android campaign\",\"~channel\":\"Messenger\",\"~creation_source\":2,\"~feature\":\"mySharefeature2\",\"~id\":\"548709306829611246\",\"~stage\":\"10\",\"~tags\":[\"myShareTag1\",\"myShareTag2\"]}"
}
```